molgenis-python/src/main/resources/python/molgenis.py

```diff
@@ -0,0 +1,72 @@
+import requests;
+import json;
+import urllib;
+
+class Connection():
+	def __init__(self, url):
+	    self.url = url
+
+	def login(self, username, password):
+		response = requests.post(self.url + "v1/login", 
+			data=json.dumps({"username":username, 'password':password}), 
+			headers={"Content-Type":"application/json"})
+		if response.status_code == 200:
+			self.token = response.json()["token"]
+		return response
+
+	def _getTokenHeader(self):
+		try:
+			return {"x-molgenis-token":self.token}
+		except AttributeError:
+			return {}
+
+	def _getTokenHeaderWithContentType(self):
+		headers = self._getTokenHeader()
+		headers.update({"Content-Type":"application/json"})
+		return headers
+
+	def logout(self):
+		response = requests.post(self.url + "v1/logout",
+			headers = self._getTokenHeader())
+		if response.status_code == 200:
+			self.token = None
+		return response
+
+	def get(self, entity, q=None, attributes=None, num = 1000, start = 0, sortColumn= None, sortOrder = None):
+		if q:
+			response = requests.post(self.url + "v1/" + urllib.quote_plus(entity),
+				headers = self._getTokenHeaderWithContentType(),
```

Comment:
Is this planned to be for Python 2 or 3? In Python 3 it is urllib.parse.quote_plus()